jinja2/base.html

```diff
@@ -158,10 +166,14 @@
     {% javascript 'main' %}
     <script>
     if (window.mdn && mdn.analytics) mdn.analytics.trackOutboundLinks();
-</script>
+  </script>
     {% for script in scripts %}
       {% javascript script %}
     {% endfor %}
+    {% if settings.MDN_CONTRIBUTION %}
```

Comment:
We also need to invert this logic:

* No stripe JS by default
* Include stripe JS on homepage, wiki doc page, contribute subpages